backend/src/services/companyFiles.service.ts

```diff
@@ -0,0 +1,118 @@
+import CompanyFile from '../models/companyFiles.model'
+import * as CompanyService from './company.service'
+import { s3 } from '../configs/aws.config'
+import { v4 as uuidv4 } from 'uuid'
+
+export type CompanyImageType = {
+  companyId: string
+  fileUrl: string
+  fileDescription: 'Imagen'
+  fileFormat?: string
+}
+
+/**
+ * @brief
+ * Función del servicio sube una imagen a la base de datos
+ * @param CompanyImage Objeto con la información de la imagen
+ * @returns Promesa con la información de la imagen o null
+ */
+export const uploadCompanyImage = async (
+  companyImage: CompanyImageType
+): Promise<CompanyFile | null> => {
+  return await CompanyFile.create(companyImage)
+}
+
+type FileDescription =
+  | 'INE representante legal'
+  | 'Acta constitutiva'
+  | 'Curriculum'
+  | 'Directorio de instaladores certificados de CDMX'
+  | 'Padron de empresas especializadas FIDE'
+  | 'Certificaciones sistemas fotovoltaicos'
+  | 'NOM-027-ENER/SCH-2018'
+  | 'NMX-ES-004-NORMEX-2015'
+  | 'Archivos presion mayor a 294k Pa'
+  | 'Archivos presion menor a 294k Pa'
+  | 'Carta de compromiso'
+  | 'Imagen'
+  | 'Otro' // EVITAR USAR ESTE VALOR
```

Comment:
Delete the OTHER option if it is not needed

Reply:
its needed